Src/FluentAssertions/Primitives/StringWildcardMatchingStrategy.cs

```diff
@@ -50,7 +50,17 @@ private static string ConvertWildcardToRegEx(string wildcardExpression)
 
     private string CleanNewLines(string input)
     {
-        return IgnoreNewLineDifferences ? input.RemoveNewLines() : input;
+        if (IgnoreAllNewlines)
+        {
+            return input.RemoveNewLines();
+        }
+
+        if (IgnoreNewlineStyle)
+        {
+            return input.RemoveNewlineStyle();
+        }
```

Comment:
Can it happen that both `IgnoreAllNewlines` and `IgnoreNewlineStyle` are set?

Reply:
Currently all instances of `StringWildcardMatchingStrategy` only use one of the two.
I wanted to have as little impact on the existing functionality as possible, so I kept the two boolean flags.
One all newlines are removed, the newline style has no impact, so I don't have to apply both in `CleanNewLines`